### PR TITLE
Multiple-table

### PR DESCRIPTION
**Pull request by LoviMadaan**
### In this pull request, Implement multiple tables in the vet_clinic database following these steps:

- [ ] Create a table named owners with the following columns:

-  id: integer (set it as autoincremented PRIMARY KEY)
-  full_name: string
-  age: integer

- [ ]  Create a table named species with the following columns:

-  id: integer (set it as autoincremented PRIMARY KEY)
-  name: string

- [ ]  Modify animals table:

 Make sure that id is set as autoincremented PRIMARY KEY

-  Remove column species
-  Add column species_id which is a foreign key referencing species table
-  Add column owner_id which is a foreign key referencing the owners' table
-  Remember all this goes in schema.sql file.

- [ ]  Insert the following data into the owners table:

-  Sam Smith 34 years old.
-  Jennifer Orwell 19 years old.
-  Bob 45 years old.
-  Melody Pond 77 years old.
-  Dean Winchester 14 years old.
-  Jodie Whittaker 38 years old.

 

- [ ] Insert the following data into the species table:

-  Pokemon
-  Digimon

- [ ]  Modify your inserted animals so it includes the species_id value:

-  If the name ends in "mon" it will be Digimon
-  All other animals are Pokemon

 

- [ ] Modify your inserted animals to include owner information (owner_id):

-  Sam Smith owns Agumon.
-  Jennifer Orwell owns Gabumon and Pikachu.
-  Bob owns Devimon and Plantmon.
-  Melody Pond owns Charmander, Squirtle, and Blossom.
-  Dean Winchester owns Angemon and Boarmon.
-  Remember these insertions and modifications happen in data.sql file.

-  Write queries (using JOIN) to answer the following questions:

-  What animals belong to Melody Pond?
-  List of all animals that are pokemon (their type is Pokemon).
-  List all owners and their animals, remember to include those that don't own any animal.
-  How many animals are there per species?
-  List all Digimon owned by Jennifer Orwell.
-  List all animals owned by Dean Winchester that haven't tried to escape.
-  Who owns the most animals?
-  Remember all these should be written in queries.sql file.

- [ ] 
Screenshot 2023-05-20 160606
Screenshot 2023-05-20 160538
Screenshot 2023-05-20 160459
Screenshot 2023-05-20 160321
Screenshot 2023-05-20 160252
Screenshot 2023-05-20 160206
Screenshot 2023-05-20 152357
Take a screenshot of the results of your queries.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,99 @@
...
+SELECT a.name
+FROM animals a
+JOIN species s ON a.species_id = s.id
+JOIN owners o ON a.owner_id = o.id
+WHERE s.name = 'Digimon' AND o.full_name = 'Jennifer Orwell';
+
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Dean Winchester';
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Good job working on this query, but this is just all the animals owned by `Dean Winchester`. Please kindly write a query to list all animals owned by Dean Winchester that haven't tried to escape.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @LoviMadaan , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AnselemOdims — COMMENTED** [review]
# Invalid Request 🛑 
You have been doing very good work on this project but unfortunately, I can not proceed with the review as there are some discrepancies. Check below for details.

- Please kindly ensure that you use the correct Gitflow when opening pull request

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by topeogunleye — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @LoviMadaan,  

Overall, great job on your code! You have successfully met the project requirements and implemented the necessary queries to retrieve the required information. Your code demonstrates a good understanding of SQL and proper usage of JOIN operations.Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 

1. Query Accuracy:
   - Your queries accurately retrieve the desired data, addressing the specific requirements of each query.
   - You have used JOIN operations effectively to combine the necessary tables and retrieve the relevant information.

2. Code Readability:
   - Your code is well-structured and easy to follow, with clear table aliases and concise syntax.
   - The use of meaningful variable and column names enhances code readability.

3. Handling Data Relationships:
   - You have successfully established and utilized the relationships between tables, such as owner_id and species_id, to retrieve the required data.
   - Your JOIN operations effectively connect the related tables based on their corresponding IDs.

4. Comprehensive Testing:
   - Your queries cover various scenarios, including retrieving animals by species, owners and their animals (including those without any animals), and counting animals per species.
   - You have correctly handled filtering conditions, such as selecting Digimon owned by Jennifer Orwell or animals owned by Dean Winchester with no escape attempts.

5. Efficiency and Optimization:
   - Your code executes efficiently, leveraging the power of JOIN operations to combine tables and retrieve the necessary information in a single query.


Overall, your code demonstrates a strong understanding of SQL concepts, and your implementation aligns well with the project requirements. Keep up the good work, and feel free to reach out if you have any further questions or need additional guidance.

## Suggested Improvement (Optional):
In future pull requests, consider following the recommended branch naming conventions, such as using branch names that reflect the specific feature or task being implemented. This will enhance code organization and collaboration within the team. For example: `feature/added-database-table`.

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**